src/Native/Runtime/unix/UnixContext.cpp

```diff
@@ -301,7 +305,7 @@ bool GetUnwindProcInfo(PCODE ip, unw_proc_info_t *procInfo)
     ((uint32_t*)(unwContext.data))[32] = ip;
 #elif _WASM_
     ASSERT(false);
-#else
+#elif !defined(_X86_)
```

Comment:
I would be nice to add `ASSERT(false)` for x86 to save somebody some debugging time in future.